.nvmrc

```diff
@@ -0,0 +1 @@
+v14.19.3
```

Comment:
@thomasfava moet dit tot op de minor versie zijn of kunnen we hiervoor alleen de major specifiëren?

Reply:
Enkel volledige versie nummers zijn toegelaten in nvmrc files.